keyboards/kbdfans/d45/config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER 0x0001
+#define PRODUCT D45
```

Comment:
At the very least, `PRODUCT` needs to be quoted:
```suggestion
#define DEVICE_VER 0x0001
#define PRODUCT "D45"
```
In the near future, the USB information needs to be [moved from `config.h` to `info.json`](https://docs.qmk.fm/#/ChangeLog/20221126?id=usb-ids-refactoring). If you don't at least quote the keyboard name, this keyboard will stop compiling when [the changes in this branch](https://github.com/vial-kb/vial-qmk/tree/merge-2023-03-12) are merged into the `vial` branch. That merge may or may not happen before your PR is even accepted.

This general advice to migrate keyboard-level information `info.json` goes for matrix pins, diode direction, MCU and bootloader, and you should apply it to any PR you open that adds keyboards which don't exist upstream. [Please see other examples of how this works in the repository](https://github.com/vial-kb/vial-qmk/blob/vial/keyboards/viktus/minne/info.json). You can also run `qmk info -f json -kb kbdfans/d45/v1` to assist with the conversion (after you move this keyboard to the v1 folder)

Reply:
Moved to info.json along with the other bootloader/diode/mcu/lighting/etc.